modules/dnn/test/test_layers.cpp

```diff
@@ -762,7 +762,11 @@ TEST_F(Layer_RNN_Test, get_set_test)
     EXPECT_EQ(shape(outputs[1]), shape(nT, nS, nH));
 }
 
+#ifndef OPENCV_DNN_EXTERNAL_PROTOBUF
 TEST_P(Test_Caffe_layers, Accum)
+#else
+TEST_P(Test_Caffe_layers, DISABLED_Accum)  // requires patched protobuf (available in OpenCV source tree only)
+#endif
```

Comment:
Skip exception is more preferable:
```cpp
TEST_P(Test_Caffe_layers, Accum)
{
#ifdef
  throw SkipTestException("Requires patched protobuf");
#else
    if (backend == DNN_BACKEND_OPENCV && target != DNN_TARGET_CPU)
        applyTestTag(CV_TEST_TAG_DNN_SKIP_OPENCL, CV_TEST_TAG_DNN_SKIP_OPENCL_FP16);

    testLayerUsingCaffeModels("accum", false, false, 0.0, 0.0, 2);
    testLayerUsingCaffeModels("accum_ref", false, false, 0.0, 0.0, 2);
#endif
}

Reply:
Thank you for review!
I implemented new patch for some layers to disable, which is same as the already implemented tests for `interp`.
Is my recognition that the test for Interp should be modified in the same way correct?

https://github.com/opencv/opencv/blob/590f150d5e032165e27d81294c9b7ac710b77f11/modules/dnn/test/test_layers.cpp#L1654-L1658

Reply:
Yes, please. It is just a convenient way.